dev/how-to/deploy/orchestrated/ansible.md

```diff
@@ -59,15 +59,19 @@ Log in to the Control Machine using the `root` user account, and run the corresp
 
 - If you use a Control Machine installed with CentOS 7, run the following command:
 
-    ```
-    # yum -y install epel-release git curl sshpass
-    # yum -y install python-pip
+    {{< copyable "shell-root" >}}
+
+    ```shell
+    yum -y install epel-release git curl sshpass && \
+    yum -y install python2-pip
```

Comment:
@liubo0127 I modified `python-pip` to `python2-pip` for all docs versions according to [that in the Chinese version](https://pingcap.com/docs-cn/stable/how-to/deploy/orchestrated/ansible/#%E5%9C%A8%E4%B8%AD%E6%8E%A7%E6%9C%BA%E4%B8%8A%E5%AE%89%E8%A3%85%E7%B3%BB%E7%BB%9F%E4%BE%9D%E8%B5%96%E5%8C%85).

Reply:
LGTM